### DISCUSSION TITLE
0.9.5 Snapshot

### DISCUSSION THREAD

**Discussion by mariuz** [mariuz/flamerobin#288] [General]
## What's Changed
* Style Configurator. by @Jdochoa in https://github.com/mariuz/flamerobin/pull/284
* Assertion failed by @Jdochoa in https://github.com/mariuz/flamerobin/pull/287

**Full Changelog**: https://github.com/mariuz/flamerobin/compare/0.9.4...0.9.5

<hr /><em>This discussion was created from the release <a href='https://github.com/mariuz/flamerobin/releases/tag/0.9.5'>0.9.5 Snapshot</a>.</em>

**Comment by mrjohnr** [comment]
hi, good to have build files too
